kube-rbac-proxy/test.yaml

```diff
@@ -0,0 +1,9 @@
+# ---
+# tests:
+#   root_user_test: True
+#   default_user: kube-rbac-proxy
+#   users_config:
+#     - name: kube-rbac-proxy
+#       group: kube-rbac-proxy
+#       gid: 65533
+#       uid: 65533
```

Comment:
Why are these tests commented out? If they're not working we need to fix the tests or fix the Dockerfile.